docs/config/01-overview.md

```diff
@@ -273,3 +273,37 @@ taskQueue: 'async'
 ## testing
 
 Please see the [testing config docs](/testing-config).
+
+## transformAliasedImportPaths
```

Comment:
Once we land this flag/feature, do you think we should add test path alias transformations to this flag, or introduce another one?  I'm wondering if we should add a comment about these paths for spec tests in this section a) in general, b) for now, c) not at all. Thoughts?

Reply:
My instinct would be to keep it simple and not have a separate flag for test behavior, so that if you set this to `true` you'd get the transformer running on spec tests as well.

I was also hoping to land the change for spec tests in 3.1 alongside this change, but I can add a small 'info' thingy now saying that this doesn't yet apply to spec tests in case we end up shipping the change before the patch for spec tests lands.

Reply:
> I was also hoping to land the change for spec tests in 3.1 alongside this change, but I can add a small 'info' thingy now saying that this doesn't yet apply to spec tests in case we end up shipping the change before the patch for spec tests lands.

Yeah, that SGTM 👍